libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -426,7 +426,7 @@ open class Node<V : RibView> @VisibleForTesting internal constructor(
 
     inline fun <reified P> pluginRoot(): P? {
         var found: P? = null
-        var current: Node<*>? = parent
+        var current: Node<*>? = parent ?: this
```

Comment:
An if (isRoot) based approach would make it explicit what we're trying to do here.

But in the end if the current node is root, then the plugin which calls this method on its own node will get back a reference to itself. I'd probably rather not do that.